browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -90,6 +90,8 @@ const PrefsUtil::TypedPrefMap& BravePrefsUtil::GetWhitelistedKeys() {
         settings_api::PrefType::PREF_TYPE_BOOLEAN;
   (*s_brave_whitelist)[kNewTabPageShowClock] =
         settings_api::PrefType::PREF_TYPE_BOOLEAN;
+  (*s_brave_whitelist)[kNewTabPageClockFormat] =
```

Comment:
FYI, only need to add this if we want the setting to show up on the Settings page via `chrome.settingsPrivate` API. We just don't happen to use that API on NTP, but we do on Shields popup panel 🤷 .  No issue to keep in though.

Reply:
Removed 😄